### PR TITLE
add PyCon APAC 2015 data

### LINKED ISSUE #112: pycon-apac-2015 has no videos
There are no videos in the pycon-apac-2015 directory. I'm not sure why offhand.

This issue covers getting the video data for that conference.

### PR DESCRIPTION
**Pull request by piotrbulinski**
Resolves #112

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -48,3 +48,4 @@ docs/_build/
 # Editor things
 *.swp
 *~
+.idea
```
Comment (willkg) [inline .gitignore:51]: What're `.idea` files from?

File: data/pycon-apac-2015/videos/marsface-project-detecting-pseudo-artificial-pycon.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "R2 DAY2-04 Marsface Project: Detecting Pseudo-artificial Structures on Mars\nSpeaker: Hideki Tanaka\n\nThe discovery of \"Face on Mars\" by NASA in 1976 has attracted widespread popular and scientific interest over the past few decades. In this project, we apply various object detection techniques, such as conventional haar-like feature based methods and state-of-the-art deep learning methods, in order to detect such pseudo-artificial structures on Mars and other planets. The scientific Python ecosystem plays a crucial role in the exploration.\n\nAbout the speaker\n\nHideki Tanaka is a PyData.Tokyo organizer, optical design engineer, photographer based in Tokyo. He is a machine learning and deep learning enthusiast who is addicted to data science competitions.\n\n個人網頁連結 https://www.facebook.com/HidekiPhoto\nTwitter @atelierhide\n組織/公司 PyData.Tokyo\n頭銜 Organizer\n\nhttps://tw.pycon.org/2015apac/zh/program/71",
+  "title": "Marsface Project: Detecting Pseudo-artificial Structures ... - Hideki Tanaka (PyCon APAC 2015)",
+  "thumbnail_url": "https://i.ytimg.com/vi/XsinyMyoZPU/sddefault.jpg",
+  "quality_notes": "hd",
```
Comment (willkg) [inline data/pycon-apac-2015/videos/marsface-project-detecting-pseudo-artificial-pycon.json:10]: We're using `quality_notes` to denote issues with the quality. Things like "no sound", "picture cuts out 3:15 in", etc.

File: data/pycon-apac-2015/videos/marsface-project-detecting-pseudo-artificial-pycon.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "tags": [],
+  "duration": 2297,
+  "copyright_text": "Standard YouTube Licence",
+  "language": "en",
```
Comment (willkg) [inline data/pycon-apac-2015/videos/marsface-project-detecting-pseudo-artificial-pycon.json:14]: Currently we're using full names like "English".

**Comment by willkg** [comment]
The checks failed originally because of something I did last week--totally not your fault. I fixed the issue today and reran the checks and they all pass now.

There are a few data things in this PR that should get fixed. Also, I noticed that the `.json` files are in an odd order which I think suggests you put this together by hand and didn't use the clive library.

I'll land this and throw together a script that fixes the data issues and reorders everything. That's probably the easiest thing to do.

You're right on in that the documentation for adding videos is pretty sparse. We're still bootstrapping this project. That work is covered in issue #9.

Thank you for doing this! I really appreciate it!

File: .gitignore (hunk 1 above)
Comment (piotrbulinski) [inline .gitignore:51]: PyCharm

File: data/pycon-apac-2015/videos/marsface-project-detecting-pseudo-artificial-pycon.json (hunk 2 above)
Comment (piotrbulinski) [inline data/pycon-apac-2015/videos/marsface-project-detecting-pseudo-artificial-pycon.json:10]: understood

**Pull request merged**